recipe/patches/0001-unvendor-lemon-spdlog.patch

```diff
@@ -0,0 +1,43 @@
+From baf410ffd928054fbaa2bb27975c8466b7df9ea1 Mon Sep 17 00:00:00 2001
+From: Lorenzo Pirritano <lpirritano@anaconda.com>
+Date: Thu, 20 Jun 2024 21:07:21 +0200
+Subject: [PATCH 1/2] unvendor lemon spdlog
+
+---
+ CMakeLists.txt | 20 +++-----------------
+ 1 file changed, 3 insertions(+), 17 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 16b91f2..d94871f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -117,23 +117,9 @@ message(STATUS "Build Dotnet: ${BUILD_DOTNET}")
+ 
+ include(FetchContent)
+ set(FETCHCONTENT_QUIET OFF)
+-set(LEMON_BUILD_TESTING OFF)
+-# Remove shared libs for LEMON
+-set(BUILD_SHARED_LIBS_OLD ${BUILD_SHARED_LIBS})
+-set(BUILD_SHARED_LIBS OFF)
+-FetchContent_Declare(
```

Comment:
I think you could have set `FETCHCONTENT_TRY_FIND_PACKAGE_MODE=ALWAYS` and it would force cmake to use `find_package` instead of building the package. `FETCHCONTENT_TRY_FIND_PACKAGE_MODE` was added in cmake 2.24. And you can also set `CMAKE_REQUIRE_FIND_PACKAGE_LEMON=ON` to make sure that find_package will fail if it's not found.

The same applies for spdlog.

* https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_TRY_FIND_PACKAGE_MODE
* https://cmake.org/cmake/help/latest/variable/CMAKE_REQUIRE_FIND_PACKAGE_PackageName.html